_includes/pbc/all/install-features/202403.0/install-glue-api/install-the-customer-account-management-glue-api.md

```diff
@@ -0,0 +1,1027 @@
+
+```bash
+console data:import glossary
+```
+
+{% info_block warningBox "Verification" %}
+
+Make sure that in the database the configured data has been added to the `spy_glossary` table.
+
+{% endinfo_block %}
+
+## 5) Set up behavior
+
+Set up the following behaviors.
+
+### Migrate data to the database
+
+Generate UUIDs for the existing entities in the `spy_customer_address` database table:
+
+```sql
+console uuid:generate Customer spy_customer_address
+```
+
+
+{% info_block warningBox "Verification" %}
+
+Ensure that, in the `spy_customer_address` table, the `UUID` field has been populated for all the records:
+
+1. Run the SQL query:
+
+   ```sql
+   SELECT COUNT(*) FROM spy_customer_address WHERE uuid IS NULL;
+   ```
+
+2. Check that the result is 0 records.
+
+{% endinfo_block %}
+
+### Enable Jenkins to find and delete expired refresh tokens
+
+Set up the following configuration:
+
+**config/Zed/cronjobs/jenkins.php**
+
+```php
+<?php
+
+/**
+ * Notes:
+ *
+ * - jobs[]['name'] must not contains spaces or any other characters, that have to be urlencode()'d
+ * - jobs[]['role'] default value is 'admin'
+ */
+$stores = require(APPLICATION_ROOT_DIR . '/config/Shared/stores.php');
+$allStores = array_keys($stores);
+
+...
+
+
+/* Oauth */
+$jobs[] = [
+    'name' => 'remove-expired-refresh-tokens',
+    'command' => '$PHP_BIN vendor/bin/console oauth:refresh-token:remove-expired',
+    'schedule' => '*/5 * * * *',
+    'enable' => true,
+    'stores' => $allStores,
+];
+```
+
+### Enable resources and relationships
+
+{% info_block infoBox %}
+
+For information on how to protect resources, see [Security and authentication](/docs/scos/dev/glue-api-guides/{{page.version}}/security-and-authentication.html).
+
+{% endinfo_block %}
+
+Enable resources and relationships as follows:
+
+1. Activate the following plugins:
+
+| PLUGIN                                                  | SPECIFICATION                                                                                                                                                                                       | PREREQUISITES                                      | NAMESPACE                                                          |
+|---------------------------------------------------------|-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|----------------------------------------------------|--------------------------------------------------------------------|
+| SetCustomerBeforeActionPlugin                           | Adds customer data to the session.                                                                                                                                                                  | Expects the user field to be set in REST requests. | Spryker\Glue\CustomersRestApi\Plugin                               |
```

Comment:
Missing plugins:
* From the folder: Bundles/Oauth/src/Spryker/Zed/Oauth/Communication/Console

Reply:
This folder contains only console commands, are you sure the path is correct?